app/javascript/lib/study-overview.js

```diff
@@ -179,36 +167,47 @@ function renderScatterPlot(target, rawPlot) {
       $(`#${plotId}`).data('camera', newCamera)
     }
   })
-
-  window.SCP.scatterPlotLayout = layout
 }
 
-/** Fetch and draw scatter plot for default Explore tab view */
-async function drawScatterPlot() {
-  const spinnerTarget = $('#plots')[0]
+/** draws scatter plot */
+async function drawScatterPlot(accession, cluster, plotIndex) {
+  const plotId = `cluster-plot-${plotIndex}`
+  const legendId = `cluster-legend-${plotIndex}`
+
+  let plotClass = '' // For only 1 plot (study without spatial data)
+  if (window.SCP.numPlots > 1) {
+    plotClass = ' plot-left'
+    if (plotIndex !== 0) {
+      plotClass = ' plot-right'
+    }
+  }
```

Comment:
I think this assigning of CSS classes could be avoided by using the :first-of-type selector to accomplish the styling you want

Reply:
Nice!  I did essentially this in 555d1f9.